resources/assets/js/store/Assessment/assessmentSelector.ts

```diff
@@ -1,151 +1,212 @@
-import isEqual from "lodash/isEqual";
+import { isEqual } from "lodash";
```

Comment:
Make sure to import the functions directly from lodash:
```js
import isEqual from "lodash/isEqual";
```
that case in particular saves about 65kb.

Reply:
08ab54cb35be20b3de917d97c1bf859c40799db4